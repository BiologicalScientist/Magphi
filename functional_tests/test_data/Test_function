This file
Is
A
test
