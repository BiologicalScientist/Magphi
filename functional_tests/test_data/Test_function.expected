This file
Is
A
test
Fail!